### ISSUE TITLE
[Animepahe] Double episodes not named correctly (Detective Conan)

### ISSUE THREAD

**Issue by marcel2789** [SenZmaKi/Senpwai#57]
When I'm downloading episodes 113-120 from Detective Conan, episode 116-117 (that's how it's named on animepahe) gets named `Meitantei Conan E116.mp4`. This causes the next episode names to be shifted by one. E.g. episode 118 is named episode 117.

This Problem gets worse if more double episodes are in the download range.

![image](https://github.com/user-attachments/assets/1b863b7f-b372-4d78-8504-8b7cb4ff1192)

**Comment by SenZmaKi** [comment]
Yep this is a known issue, would you say naming the episode 117 is better than 116 cause 116-177 will break things

**Comment by marcel2789** [comment]
I think it would be great to add an option in the settings and let the user decide.

The naming depends on the use case. I'm using VLC on my tablet and it doesn't care if the episodes are named 116-117. But I also agree that some software like kodi (I haven't tested that) may not like this format.

**Comment by SenZmaKi** [comment]
yeah letting the user decide is too much of a pain the ass, I'll just figure something out

**Comment by candrapersada** [comment]
i have a problem with episode 13.5
https://animepahe.ru/play/a71d8482-af1b-2bf5-be61-bc33439d97d5/0e3bfed917a2a36865cf19d2a01737f919bfe620e11561162e8c6bfd0a7a93b7
from episode 14 it became episode 15 that was downloaded?

**Comment by candrapersada** [comment]
how to get Detective Conan, episodes 1150-1151?
